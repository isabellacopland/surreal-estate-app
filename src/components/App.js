import "../styles/app.css";
import React from "react";
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
import Navbar from "./Navbar";
import Properties from "./Properties";
import AddProperty from "./AddProperty";

const App = () => {
  return (
    <Router>
      <div className="App">
        <Navbar />
        <Switch>
          <Route exact path="/" component={Properties} />
          <Route exact path="/add-property" component={AddProperty} />
        </Switch>
      </div>
    </Router>
  );
};

export default App;
