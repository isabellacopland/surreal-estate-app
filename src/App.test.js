import React from "react";
import { render } from "@testing-library/react";
import App from "../src/components/App";

describe("App", () => {
  test("renders App", () => {
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
  });
});