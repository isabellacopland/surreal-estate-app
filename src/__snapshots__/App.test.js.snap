// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders App 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <img
        alt="surreallogo"
        class="navbarlogo"
        src="logo.png"
      />
      <ul
        class="navbar-links"
      >
        <a
          class="navbar-links-item"
          href="/"
        >
          View Properties
        </a>
        <a
          class="navbar-links-item"
          href="/add-property"
        >
          Add a Property
        </a>
      </ul>
    </div>
    <div
      class="properties"
    >
      <h2>
        Properties
      </h2>
    </div>
  </div>
</DocumentFragment>
`;
